shared/build.gradle.kts

```diff
@@ -5,10 +5,12 @@ plugins {
 }
 
 kotlin {
+    targetHierarchy.default()
+
     android {
         compilations.all {
             kotlinOptions {
-                jvmTarget = "1.8"
+                jvmTarget = "17"
```

Comment:
jvmTarget = "1.8" is required for android

Reply:
Got it, reverted